js/renovation/ui/scheduler/view_model/to_test/views/utils/const.ts

```diff
@@ -0,0 +1,20 @@
+import { ViewType } from '../../../../types';
+
+export const VIEWS: Record<string, ViewType> = {
+  DAY: 'day',
+  WEEK: 'week',
+  WORK_WEEK: 'workWeek',
+  MONTH: 'month',
+  TIMELINE_DAY: 'timelineDay',
+  TIMELINE_WEEK: 'timelineWeek',
+  TIMELINE_WORK_WEEK: 'timelineWorkWeek',
+  TIMELINE_MONTH: 'timelineMonth',
+  AGENDA: 'agenda',
+};
+
+export const TIMELINE_VIEWS: ViewType[] = [
```

Comment:
Minor comment.

Maybe, we shall use Set or Record<ViewType, boolean> instead of the array here?
Connected with the previous comment.